fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/impl/ResourceReferenceDAO.java

```diff
@@ -426,4 +426,25 @@ public void persist(Collection<ResourceTokenValueRec> records) {
             insertResourceTokenRefs(entry.getKey(), entry.getValue());
         }
     }
-}
+
+    @Override
+    public List<Long> readCommonTokenValueIdList(String tokenValue) {
+        List<Long> result = new ArrayList<>();
+        final String SQL = ""
+                + "SELECT c.common_token_value_id "
+                + "  FROM common_token_values c "
+                + " WHERE c.token_value = ?";
+        try (PreparedStatement ps = connection.prepareStatement(SQL)) {
```

Comment:
QueryUtil for the prepared statement?  
Also is the intento to eventually replace with the database-utils model?

Reply:
Eventually we should do all our SQL work using the new Select object.